SYCL/ESIMD/api/functional/common.hpp

```diff
@@ -14,15 +14,15 @@
 
 #pragma once
 
-#include <sycl/ext/intel/experimental/esimd.hpp>
-#include <sycl/sycl.hpp>
-
 #include "../../esimd_test_utils.hpp"
 #include "logger.hpp"
 #include "type_coverage.hpp"
 #include "type_traits.hpp"
 #include "value.hpp"
 
+#include <sycl/ext/intel/experimental/esimd.hpp>
+#include <sycl/sycl.hpp>
+
 #include <vector>
```

Comment:
Why such move of includes? By doing so we might have something from our headers affecting the esimd api itself.

Reply:
I've done it [here](https://github.com/intel/llvm-test-suite/pull/762/commits/afbf175e94a84d6a621ba620d2c0119abd0ec6aa#diff-ef0766a5c874c909fd25e4eec71893c4614c91112485144e4efcdab4d856d0a0R17-R26)